### PR TITLE
Update formatting of media cards on the feed page

### PR DESCRIPTION
**Pull request by jordansimsmith**
GitHub Issue (If applicable): #50 

## PR Type

What kind of change does this PR introduce?

- Feature
- Code style update (formatting)

## What is the current behavior?

Cards hard coded and not properly formatted on feed page (just there for testing).

## What is the new behavior?

- Changed single column to dynamic grid (3 columns on my monitor, will adjust depending on the size).
- All cards are the same height using flexbox
- Formatted MediaCard.jsx as the pre commit hook did not run on the previous PR (`yarn install` perhaps)
- Updated snapshot

2020-03-15-071302_2560x1440_scrot

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current upstream repo
- [ ] Added new tests to the test suite for the feature
- [x] Tested the full test suite, and ensured that the feature does not break current build.
- [ ] Docs have been added/updated which fit (for bug fixes / features)
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal)
- [ ] Checked licensing of Frameworks/Libraries (if applicable) 

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: client/src/components/MediaCard.jsx [hunk 1]
```diff
@@ -38,79 +40,85 @@ const useStyles = makeStyles(theme => ({
...
         <Typography variant="body2" color="textSecondary" component="p">
           {props.mainText}
         </Typography>
       </CardContent>
-      <CardActions style={{ backgroundColor: barColour(props.media) }} disableSpacing >
+      <CardActions
+        style={{ backgroundColor: barColour(props.media) }}
+        disableSpacing
+      >
         <img src={mediaIcon(props.media)} />
```
Comment (Matteas-Eden) [inline client/src/components/MediaCard.jsx]: Lack of an `alt` prop is bad practice, but not a blocker.

File: client/src/components/MediaCard.jsx [hunk 2]
```diff
@@ -38,79 +40,85 @@ const useStyles = makeStyles(theme => ({
...
     <Card className={classes.root}>
       <CardHeader
         avatar={
-          <Avatar alt={props.username} src={props.avatarLink ?
-            props.avatarLink :
-            "https://img.icons8.com/windows/64/000000/user.png"} className={classes.avatar}
+          <Avatar
+            alt={props.username}
+            src={
+              props.avatarLink
```
Comment (HarrisonLeach1) [inline client/src/components/MediaCard.jsx]: To make this code more succinct you could do:
```javascript
    props.avatarLink || 'https://img.icons8.com/windows/64/000000/user.png'
```

File: client/src/components/MediaCard.jsx [hunk 3]
```diff
@@ -38,79 +40,85 @@ const useStyles = makeStyles(theme => ({
...
             <MoreVertIcon />
           </IconButton>
         }
         title={props.title}
-        subheader={props.username + " - " + props.relativeTime}
+        subheader={props.username + ' - ' + props.relativeTime}
       />
-      {props.imageLink ?
+      {props.imageLink ? (
         <CardMedia className={classes.media} image={props.imageLink} />
```
Comment (HarrisonLeach1) [inline client/src/components/MediaCard.jsx]: This code could be made more concise using the following:
```javascript
      {props.imageLink &&
        <CardMedia className={classes.media} image={props.imageLink} />}
```

File: client/src/components/MediaCard.jsx (hunk 2 above)
Comment (Matteas-Eden) [reply client/src/components/MediaCard.jsx]: Sorry, hijacking the PR, how does that work?

Comment (jordansimsmith) [reply client/src/components/MediaCard.jsx]:
```
let a = left || right
```

if `left` is truthy, assign it to `a`. Otherwise assign `right` to `a`.

Comment (jordansimsmith) [reply client/src/components/MediaCard.jsx]: @HarrisonLeach1 I've refactored the component in db3db1c

File: client/src/components/MediaCard.jsx (hunk 3 above)
Comment (jordansimsmith) [reply client/src/components/MediaCard.jsx]: Updated in db3db1c

File: client/src/components/MediaCard.jsx (hunk 1 above)
Comment (jordansimsmith) [reply client/src/components/MediaCard.jsx]: Updated in db3db1c

File: client/src/components/MediaCard.jsx (hunk 2 above)
Comment (HarrisonLeach1) [reply client/src/components/MediaCard.jsx]: If `props.avatarLink` has a falsy value. e.g. `false`, `null` or `undefined` much like other languages js will look to the right-hand side value to see if it is true (or truthy in javascript). In this case the right side is always truthy, but in js the true will not be returned but the truthy value so `'https://img.icons8.com/windows/64/000000/user.png'` is returned.

Comment (HarrisonLeach1) [reply client/src/components/MediaCard.jsx]: For more information see [here](https://dev.to/acroynon/javascript-or-assignment-operator-1e8g)

**Pull request merged**